README.md

```diff
@@ -35,3 +35,34 @@ oc adm must-gather --image=registry/username/your-built-image
 
 ## Obfuscate confidential information
 There is a dedicated effort to obfuscate and omit confidential information. Head over to [openshift/must-gather-clean](https://github.com/openshift/must-gather-clean) for more information.
+
+## Tests
```

Comment:
do we have any tests defined for this repository ?

if not do we need to introduce ote compatible binary ?